core/tests/test_utils.py

```diff
@@ -250,19 +280,21 @@ def get_csrf_token_from_response(self, response):
     def signup(self, email, username):
         """Complete the signup process for the user with the given username."""
         self.login(email)
-
-        response = self.testapp.get(feconf.SIGNUP_URL)
-        self.assertEqual(response.status_int, 200)
-        csrf_token = self.get_csrf_token_from_response(response)
-        response = self.testapp.post(feconf.SIGNUP_DATA_URL, {
-            'csrf_token': csrf_token,
-            'payload': json.dumps({
-                'username': username,
-                'agreed_to_terms': True
+        # Signup uses a custom urlfetch mock (URLFetchServiceMock), instead
+        # of the stub provided by testbed. This custom mock is disabled
```

Comment:
Say here that this is to avoid external calls being made to Gravatar when running the backend tests.